src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -1092,6 +1092,29 @@ protected static ImmutableArray<string> CheckTupleElementNames(int cardinality,
 
         #endregion
 
+        #region Operations
+
+        /// <summary>
+        /// Gets the low-level operation corresponding to the method's body.
+        /// </summary>
+        /// <param name="method">The method symbol.</param>
+        /// <param name="cancellationToken">An optional cancellation token.</param>
+        /// <returns>The low-level operation corresponding to the method's body.</returns>
+        internal IOperation GetOperation(IMethodSymbol method, CancellationToken cancellationToken = default)
```

Comment:
>GetOperation [](start = 28, length = 12)

Should also test with members of compiler synthesized types: anonymous types (both public and private anonymous type symbols), tuple types, etc.